roles/commcare_sync/tasks/django.yml

```diff
@@ -64,6 +64,10 @@
     - "{{ static_dir }}"
     - "{{ media_dir }}"
 
+- name: set acl on app log directory.
+  shell: setfacl -d -m u::rwx,g::rw,o::rw {{ item }}
```

Comment:
does the user need to have the executable flag set? Is that the standard default?

Reply:
Also could we make them globally writeable but not readable? Just thinking that might be safer if it works.

Reply:
yes, user/owner has full permissions on the directory. Earlier, Other's has no write permission on the new files so I had to give 'rw' to other's. This will help us in our case.